driver/ppm_events_public.h

```diff
@@ -1608,19 +1608,20 @@ enum extra_event_prog_code
 	PPM_SC_X(EPOLL_WAIT_OLD, 379) \
 	PPM_SC_X(RSEQ, 380) \
 	PPM_SC_X(CREATE_MODULE, 381) \
-	PPM_SC_X(SCHED_GETATTR, 382) \
-	PPM_SC_X(FACCESSAT2, 383) \
-	PPM_SC_X(_SYSCTL, 384) \
-	PPM_SC_X(QUERY_MODULE, 385) \
-	PPM_SC_X(GET_MEMPOLICY, 386) \
-	PPM_SC_X(SYNC_FILE_RANGE, 387) \
-	PPM_SC_X(PROCESS_MADVISE, 388) \
-	PPM_SC_X(MEMBARRIER, 389) \
-	PPM_SC_X(IOPL, 390) \
-	PPM_SC_X(CLOSE_RANGE, 391) \
-	PPM_SC_X(FANOTIFY_MARK, 392) \
-	PPM_SC_X(RECV, 393) \
-	PPM_SC_X(SEND, 394)
+	PPM_SC_X(NA_1, 382) \
```

Comment:
Added back old unused PPM_SC_PIDFD_GETFD calling it `NA_1`.
PPM_SC codes starting with `NA_` will be skipped while popilating syscall info table.